code/modules/modular_computers/file_system/programs/generic/manifest.dm

```diff
@@ -3,6 +3,7 @@
 	filedesc = "Crew Manifest"
 	program_icon_state = "generic"
 	extended_desc = "This program is used for viewing the crew manifest."
+	usage_flags = PROGRAM_ALL
```

Comment:
This is unecessery

Reply:
Imho a explicit definition of the usage_flags for the modular computer programs is useful (even in the default case) so its easy to see what flags a application has.